### PR TITLE
allow private methods on audited condition

### PR DESCRIPTION
**Pull request by rauann**
Adding `true`as a second parameter to include private methods on conditional verification.

### CODE REVIEW

File: lib/audited/auditor.rb [hunk 1]
```diff
@@ -285,11 +285,10 @@ def auditing_enabled
 
       def run_conditional_check(condition, matching: true)
         return true if condition.blank?
-
         return condition.call(self) == matching if condition.respond_to?(:call)
-        return send(condition) == matching if respond_to?(condition.to_sym)
+        return send(condition) == matching if respond_to?(condition, include_all: true)
 
-        true
+        false
```
Comment (tbrisker) [inline lib/audited/auditor.rb]: why did you change the default to false?

File: lib/audited/auditor.rb [hunk 2]
```diff
@@ -285,11 +285,10 @@ def auditing_enabled
 
       def run_conditional_check(condition, matching: true)
         return true if condition.blank?
-
         return condition.call(self) == matching if condition.respond_to?(:call)
-        return send(condition) == matching if respond_to?(condition.to_sym)
+        return send(condition) == matching if respond_to?(condition, include_all: true)
```
Comment (tbrisker) [inline lib/audited/auditor.rb]: This actually isn't a kwarg, so just `true` as the second parameter is enough. (this still works since you are creating a new hash here and passing it to the function, and since a hash is non-falsy the check in `respond_to?` still passes, but every check of this function creates an extra object allocation)

File: lib/audited/auditor.rb (hunk 1 above)
Comment (rauann) [reply lib/audited/auditor.rb]: With the second parameter as `true` on `respond_to?` at last return, the default will never be raised I think. Anyway, I'll rollback it.

File: lib/audited/auditor.rb (hunk 2 above)
Comment (rauann) [reply lib/audited/auditor.rb]: Sure, good catch. Updated.
